resources/views/app.blade.php

```diff
@@ -3,6 +3,9 @@
 <head>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0"/>
+    @if (app()->environment('production'))
```

Comment:
I think it should check if app is in **non** local environment. Because on `beta` environment we wont be able to detect any issues related to CSP header.

Reply:
yep, sure.
I will fix that